src/signals/incident-management/containers/IncidentCategoryContainer/components/Add/index.js

```diff
@@ -20,7 +21,8 @@ class Add extends React.Component { // eslint-disable-line react/prefer-stateles
   }
 
   render() {
-    const { subcategoryList } = this.props;
+    const { subcategoryList, loading } = this.props;
+    this.categoryForm.controls.forceRender.setValue(Math.random());
```

Comment:
What is the idea behind this forceRender, having a random value?

Reply:
the Child form containing the button does not rerender when the Parent is rendered with the new loading value. so I added a form control  forceRender that changes when the Parent is rendered. i agree it is Rather ugly.

Reply:
It's neater to use the loading prop for that, so something like:
`this.categoryForm.controls.loading.setValue(loading);`
instead of some random value for forceRender. Maybe add a comment to make clear why it's done.

Reply:
ja dat is eleganter. ik stop het erin